test/e2e/flask/user-operations.spec.ts

```diff
@@ -239,7 +238,10 @@ async function withAccountSnap(
         ERC_4337_ACCOUNT_SALT,
       );
 
-      await driver.closeWindow();
+      const testDapp = new TestDapp(driver);
+      await testDapp.openTestDappPage();
+      await testDapp.connectAccount({ publicAddress: ERC_4337_ACCOUNT });
```

Comment:
Why replace the fixture with this connect step?

Reply:
unable to get this test working for me locally atm with and without the change above. will have to investigate again later

Reply:
ahh. so this test was not properly testing this scenario before. The fixture `withPermissionControllerConnectedToTestDapp` included the snap generated `ERC_4337_ACCOUNT` in the eth_accounts permission before it actually was created by and added by the snap. The changes above fix that. I'll break this out into another PR as it isn't multichain related

Reply:
done here https://github.com/MetaMask/metamask-extension/pull/29675